app/views/episode_transcripts/_tab.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= turbo_frame_tag "episode-transcript-status", data: {controller: "morphdom"} do %>
+  <% status = episode_transcript_status(episode) %>
+
+  <%= tab_link_to episode_transcripts_path(episode), data: {controller: "active-link", turbo_frame: "_top"} do %>
+    <%= t("episodes.tabs.transcript") %>
+    <% unless status == "incomplete" %>
```

Comment:
it didn't look right to have an `incomplete` badge on this tab since transcript uploads are optional, but i still render all the other badges.

Reply:
Hrm... I guess I don't think this status/badge matters at all for transcripts?  At least not on the tab.

But yes, a transcript isn't required.  So an episode isn't "incomplete" if it doesn't have one.  Vs audio, which _is_ required.